pkg/asset/rbac.go

```diff
@@ -96,7 +96,7 @@ func (s *rbac) New() []*RBACItem {
 				Rules: []rbacv1.PolicyRule{
 					{
 						APIGroups: []string{
-							"autoscaling.openshift.io",
+							"admission.autoscaling.openshift.io",
```

Comment:
I think this is the same value as `s.values.AdmissionAPIGroup`, you could probably use that instead of the raw string?